package.json

```diff
@@ -18,9 +18,12 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
+    "@react-email/render": "^1.0.5",
```

Comment:
feels like this must be added as a peerDependency instead. If the client is using react-email they have to install it in their project instead.